### PR TITLE
lifeguards redo

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

**Review by ryanchou-dev — APPROVED** [review]
where do you find the time to revamp so much code!!

File: solutions/bronze/usaco-784.mdx [hunk 1]
```diff
@@ -75,138 +73,209 @@ import java.util.*;
...
+		int[] startT = new int[cows];
+		int[] endT = new int[cows];
+		int[] times = new int[1000];
+		int shiftTotal = -1;
+
+		for (int c = 0; c < cows; c++){
+			st = new StringTokenizer(br.readLine());
+			startT[c] = Integer.parseInt(st.nextToken());
+			endT[c] = Integer.parseInt(st.nextToken());
+			for (int k = startT[c]; k < endT[c]; k++){
```
Comment (envyaims) [inline solutions/bronze/usaco-784.mdx:91]: why k
since u use t in other places just use t lol

File: solutions/bronze/usaco-784.mdx [hunk 2]
```diff
@@ -75,138 +73,209 @@ import java.util.*;
...
+
+	/*
+	 * Simulate firing each lifeguard and compute the total
+	 * time covered by the remaining lifeguards
+	 */
+	int max_cover = 0;
+	for (int i = 0; i < n; i++) {
+		int time_covered = 0;
+		// Try to find a cow that can cover this interval
+		for (int v = 1; v <= max_time; v++) {
```
Comment (envyaims) [inline solutions/bronze/usaco-784.mdx:162]: v???

**Review by envyaims — APPROVED** [review]
wn

File: solutions/bronze/usaco-784.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/bronze/usaco-784.mdx:91]: this is part of the video solution...

File: solutions/bronze/usaco-784.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply solutions/bronze/usaco-784.mdx]: should work

**Pull request merged**